housekeeper/cli/add.py

```diff
@@ -157,7 +158,8 @@ def tag_cmd(context: click.Context, tags: List[str], file_id: int):
         raise click.Abort
 
     if file_id:
-        file_obj: File = store.file_(file_id)
+        file_obj = store.get_file_by_id(file_id)
```

Comment:
names args, and be explicit about id, is it an entry id, version id, internal id? etc.